aiojsonrpc/serializer/json.py

```diff
@@ -1,12 +1,21 @@
 from functools import partial
 from . import factory
 
-try:
```

Comment:
Actually no need backoff at all, since ujson is in requirements.
Probably it makes sense to remove ujson from requirements and add an ability to user to set own json module to use.